gcn/notices/core/DispersionMeasure.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/core/DispersionMeasure.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "Schema for Radio Transients Observables",
+  "description": "Dispersion Measure schema used for radio transients",
+  "properties": {
+    "dm": {
+      "type": "number",
+      "description": "Dispersion measure (DM) of the burst [pc/cm^3], representing the integrated column density of free electrons along the line of sight."
+    },
+    "dm_error": {
+      "type": "array",
+      "items": { "type": "number" },
+      "maxItems": 2,
+      "description": "Uncertainity associated with the dispersion measure [pc/cm^3, 1-sigma], with optional asymmetric uncertainty."
```

Comment:
If the asymmetric uncertainty is optional, can you please accept either an array or a number?

Reply:
Are you asking for it to be like `ra_dec_error` where you can give it either a number or an array?  The way this is written now, it can be "[1.0]" or "[-1.0,+1.0]".  If we want to do like `ra_dec_error`, there are a bunch of other core schema that will need to be redefined.